File: Examples/android/CMeSdkJava/app_rework/src/main/java/me/connect/sdk/java/sample/db/entity/Connection.java
package me.connect.sdk.java.sample.db.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Connection {
    @PrimaryKey
    public int id;

    @ColumnInfo(name = "serialized_connection")
    public String serializedConnection;

    @ColumnInfo(name = "name")
    public String name;

    @ColumnInfo(name = "icon")
    public String icon;
}

File: examples/android/MSDKSampleAppKotlin/app/src/main/java/msdk/kotlin/sample/db/entity/Connection.kt
package msdk.kotlin.sample.db.entity

import androidx.room.ColumnInfo
import androidx.room.Entity

import androidx.room.PrimaryKey


@Entity
data class Connection(
        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,

        @ColumnInfo(name = "serialized")
        var serialized: String,

        @ColumnInfo(name = "name")
        var name: String,

        @ColumnInfo(name = "icon")
        var icon: String?,

        @ColumnInfo(name = "pwdid")
        var pwDid: String
)
